extensions/email/src/main/java/org/mule/extension/email/internal/commands/EmailIdConsumerExecutor.java

```diff
@@ -44,24 +46,33 @@ void execute(Message muleMessage, Integer emailId, Consumer<Integer> consumer) {
       Object payload = muleMessage.getPayload().getValue();
       if (payload instanceof List) {
         for (Object o : (List) payload) {
-          if (o instanceof Message) {
-            emailId = getIdOrFail(((Message) o));
+          if (o instanceof OperationResult) {
```

Comment:
This changes were introduced just to maintain tests in green, this class has been deleted in the ticket that im working on now (MULE-10548)